dbt/dbt_project.yml

```diff
@@ -45,6 +45,9 @@ models:
       +schema: census
     default:
       +schema: default
+    ephemeral:
+      +schema: ephemeral
+      +materialized: ephemeral
```

Comment:
Adds a new ephemeral schema folder for _all_ future ephemeral tables. I know we kinda balked at use ephemeral models before, but `eph_final_model_long` really does feel like a good use case for one.